prisma/devSetup.ts

```diff
@@ -0,0 +1,193 @@
+/* eslint-disable func-names */
+/* eslint-disable no-console */
+import { PrismaClient } from '@prisma/client';
+import loadCSVFromFile from './loadCsv';
+
+const seedHealthcareProfessionals = async function (prisma: PrismaClient, verbose = false) {
```

Comment:
Data like professionals, specialties, and degrees should be seeded into all environments.